### ISSUE TITLE
New release/tag

### ISSUE THREAD

**Issue by GekkieHenkie** [KhronosGroup/OpenCL-ICD-Loader#111]
I understand this needs some alignment with the OpenCL-headers repository, but the current and only tag v2020.03.13 gives me troubles compiling on Ubuntu because of the multiple defined ret_val issue which got fixed two weeks later in [this pull](https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/101).

As a workaround I could return to always syncing my project to the master branch, but it feels like a step backward, since they make my project's builds unstable:
a. There is a chance the master branch of this repository and the master branch of the OpenCL-headers repository are out of sync.
b. New features, bugfixed functionality, is pushed to the master branch which gives unexpected compiler errors in my build or functionality changes in my project.

TLDR: Could you please add a new release/tag to improve stability in downstream projects? Thanks

**Comment by kpet** [comment]
Released [v2020.06.16](https://github.com/KhronosGroup/OpenCL-ICD-Loader/releases/tag/v2020.06.16)
